nameko/testing/utils.py

```diff
@@ -140,3 +140,65 @@ def find_free_port(host='127.0.0.1'):
     port = sock.getsockname()[1]
     sock.close()
     return port
+
+
+class ResourcePipeline(object):
```

Comment:
I would like to use it in nameko-sqlalchemy to speed up creation of database schemas.